scripts/color-sorting.js

```diff
@@ -0,0 +1,101 @@
+/**
+ * @fileoverview
+ * Color sorting algorithm.
+ */
+
+module.exports = {
+  sortByColors: function (colors) {
+    const isGray = (rgb, range) => {
+      var { r, g, b } = { ...rgb };
```

Comment:
Could you replace all uses of `var` by `let`? ([see this article](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Statements/let#description))

Reply:
sure.